### PR TITLE
implement exit page

### LINKED ISSUE #467: Implement exit page to godcr-fyne
Fyne interface should have a button on the tab menu that properly exits the wallet.

### PR DESCRIPTION
**Pull request by metaclips**
Screenshot 2019-12-25 at 12 12 47
closes #467

### CODE REVIEW

**Comment by oshorefueled** [comment]
The app successfully exits when I click "Yes" on the exit popup. However, when I click "No" on a page other than the overview page, the popup disappears but takes me to the overview page. The app should remain on the same page after clicking the "No" button.

File: fyne/pages/exit.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	"github.com/raedahgroup/godcr/fyne/pages/handler/values"
+	"github.com/raedahgroup/godcr/fyne/widgets"
+)
+
+func (app *AppInterface) exitPageContent() fyne.CanvasObject {
+	var popup *widget.PopUp
+
+	yesButton := widget.NewButtonWithIcon("Yes", theme.ConfirmIcon(), func() {
+		app.Window.Close()
+		fmt.Println("Exited fyne")
```
Comment (oshorefueled) [inline fyne/pages/exit.go]: This should be logged

**Review by oshorefueled — CHANGES_REQUESTED** [review]

**Comment by metaclips** [comment]
> The app successfully exits when I click "Yes" on the exit popup. However, when I click "No" on a page other than the overview page, the popup disappears but takes me to the overview page. The app should remain on the same page after clicking the "No" button.

What would be shown on exit page if the popup closes?  Showing an empty page on clicking No doesn't really seem right to be honest.

**Comment by oshorefueled** [comment]
> > The app successfully exits when I click "Yes" on the exit popup. However, when I click "No" on a page other than the overview page, the popup disappears but takes me to the overview page. The app should remain on the same page after clicking the "No" button.
> 
> What would be shown on exit page if the popup closes? Showing an empty page on clicking No doesn't really seem right to be honest.

ideally, the popup should not be a page, but just a popup capable of being visible on any page. But in a case where this isn't possible as a result of fyne's limitations, you can just go back to the previous page. It should give the same effect as if the user never left the page.

**Review by oshorefueled — APPROVED** [review]

**Pull request merged**